documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc

```diff
@@ -1,19 +1,29 @@
 [id='creating_kubernetes_secrets-{context}']
 = Creating Kubernetes Site Access Secrets
-If you run {ispn_operator} in any Kubernetes deployment (Minikube, Kind, etc.), you should create secrets that
-contain the files that allow different instances of Kubernetes to authenticate
-with each other.
+If you run {ispn_operator} in any Kubernetes deployment (Minikube, Kind, etc.), you should create secrets that contain the files that allow Kubernetes clusters to authenticate with each other.
 
-.Procedure
+Do one of the following:
 
-. Create secrets on each site that contain `ca.crt`, `client.crt`, and `client.key` from your Kubernetes installation.
+* Retrieve service account tokens from each site and then add them to secrets on each backup location, for example:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ kubectl get serviceaccounts site-a -o yaml
```

Comment:
Step should be the following
kubectl create serviceaccount site-a -n ns-site-a
kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=ns-site-a:site-a
TOKENNAME=`kubectl get serviceaccount/site-a -o jsonpath='{.secrets[0].name}' -n ns-site-a`
TOKEN=`kubectl get secret $TOKENNAME -o jsonpath='{.data.token}' -n ns-site-a | base64 --decode`

Reply:
thank you @dmvolod